docker/build_docker.sh

```diff
@@ -383,6 +383,8 @@ echo $DIFF_COMPARE > $CIRCLE_ARTIFACTS/diff_compare.txt
 echo $SCRIPT_DIR > $CIRCLE_ARTIFACTS/script_dir.txt
 echo $CURRENT_DIR > $CIRCLE_ARTIFACTS/current_dir.txt
 echo $DOCKER_INCLUDE_GREP > $CIRCLE_ARTIFACTS/docker_include_grep.txt
+touch "/tmp/image_full_name.txt"
```

Comment:
1 file separated by a comma would be better for human readability, but since we only want to support the native image only image_full_name should really be neccessary.